src/main/java/hudson/scm/IntegrityCheckinAction.java

```diff
@@ -207,8 +207,15 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
     }
 
     // Create our Integrity check-in task
-    IntegrityCheckinTask ciTask = new IntegrityCheckinTask(ciConfigPath, ciWorkspaceDir, includes,
-        excludes, build, listener, getProjectSettings(build));
+    IntegrityCheckinTask ciTask;
+	try {
+		ciTask = new IntegrityCheckinTask(ciConfigPath, ciWorkspaceDir, includes,
+		    excludes, build, listener, getProjectSettings(build));
+	} catch (Exception e) {
+		LOGGER.fine("Exception Caught!  Skipping Integrity Checkin step! :"+ e.getMessage());

```

Comment:
More helpful to use `log(Level, String, Throwable)` to keep the stack trace.

Reply:
Dear @jglick,
Noted. Will be fixed in the next release